ingestion/src/metadata/ingestion/source/database/bigquery/queries.py

```diff
@@ -63,7 +63,7 @@
   routine_name as name,
   routine_definition as definition,
   external_language as language
-FROM test_omd.INFORMATION_SCHEMA.ROUTINES
+FROM {schema_name}.INFORMATION_SCHEMA.ROUTINES
```

Comment:
@pmbrull can we try adding \`{schema_name}\` so that we don't regress on the issue like "dataset with - not found"